src/dates/format-date.js

```diff
@@ -195,4 +195,32 @@ export default function formatDate(date, format, locale = "en") {
 
         return result;
     });
+}
+
+export function formatFormat(date, format, locale = "en") {
+    if (!isDate(date)) {
+        return format;
+    }
+
+    const info = localeInfo(locale);
+    const pattern = datePattern(format, info);
+
+    return pattern.replace(dateFormatRegExp, function(match) {
```

Comment:
@ggkrustev 
The result of **datePattern** is not what I need. It returns valid pattern, but not matched to the actual date, and it is short pattern, used for replacement the symbols from it with actual data.

**datePattern** could return "**d/m/y**".
Let's say we have date 2017-04-11
Then the method replaces the 'd' with 11, 'm' with 04 and 'y' with 2017. 
So **formatDate** will return "11/4/2017"

On the other hand, I need the **dateFormatString** to return "dd/**m**/yyyy"

The same pattern applied with different date will result string with different length if the month in the above example is 10, 11 or 12. Function should return "dd/**mm**/yyyy"